source/process/docs-contributors-guide.rst

```diff
@@ -0,0 +1,35 @@
+Contributor's Guide for docs.mattermost.com
+===========================================
+
+Thank you for your interest in contributing to Mattermost documentation. Your contributions benefit thousands of organizations around the world deploying Mattermost. 
+
+To start: 
+
+Create a Ticket 
+---------------
+
+1. **Select a Help Wanted ticket** - Please select a `HELP WANTED ticket <https://github.com/mattermost/docs/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22>`_ that you'd like address with your contribution, or create a new ticket to discuss an improvement you'd like to make and wait for a project Member to approve and add the `Help Wanted` label so the change will be accepted.
+
+If your update is small and an obvious improvement, you can skip this step and submit a PR directly. 
+
+2. **Become an Approved Contributor** - Please complete the `contributor license agreement <https://www.mattermost.org/mattermost-contributor-agreement/>`_ (a.k.a. "CLA"). This is a standard form for most open source projects and it adds your the approved contributor's list for the Mattermost open source project. There are answers to frequently asked questions in the link above. 
+
+3. **Create your Pull Request** - Please create a pull request for the documentation change you would like to make. Make sure to review and follow the `Style Guide <https://docs.mattermost.com/process/sg_mattermost-doc-style.html>`_. Please include a link to the Help Wanted ticket if you have one. 
```

Comment:
We should update the style guide URL before linking to it elsewhere, I think "sg" is confusing/"not obvious".